libnavigation-core/src/main/java/com/mapbox/navigation/core/reroute/MapboxRerouteController.kt

```diff
@@ -291,6 +276,9 @@ internal class MapboxRerouteController @VisibleForTesting constructor(
                     }
                 }
             )
+            cont.invokeOnCancellation {
+                directionsSession.cancelRouteRequest(requestId)
```

Comment:
Can you please run `ev_route_refresh_parameters_for_ev_route_with_no_ev_data` locally in a loop 200-300 times and make sure it did not become flaky again? :)

Reply:
I run the test 20 times on a real device and 100 times on an emulator, all green.

Reply:
Awesome, thanks.